examples/vuejs/README.md

```diff
@@ -2,47 +2,64 @@
 <img src="https://vuejs.org/images/logo.png" width="200px"><br>
 [vuejs.org](https://vuejs.org/) | `Support Level` = `Strong`
 
-## How to build a `vue.js` component using `html5-qrcode`
+## How to create a `vue.js` component for `html5-qrcode`
 
 ### Include the js library in your project
+
 ```html
 <script src="https://unpkg.com/html5-qrcode/minified/html5-qrcode.min.js"></script>
 ```
 
-### Use a `qrcode-scanner` component
+### Create a `qrcode-scanner` component.
+
 ```js
 Vue.component('qrcode-scanner', {
-    props: {
-        qrbox: Number,
-        fps: Number,
+  props: {
+    qrbox: {
+      type: Number,
+      default: 250
     },
-    template: `<div id="qr-code-full-region"></div>`,
-    mounted: function () {
-        var $this = this;
-        var config = { fps: this.fps ? this.fps : 10 };
-        if (this.qrbox) {
-            config['qrbox'] = this.qrbox;
-        }
-
-        function onScanSuccess(qrCodeMessage) {
-            $this.$root.$emit('decodedQrCode', qrCodeMessage);
-        }
-        
-        var html5QrcodeScanner = new Html5QrcodeScanner(
-            "qr-code-full-region", config);
-        html5QrcodeScanner.render(onScanSuccess);
+    fps: {
+      type: Number,
+      default: 10
+    },
+  },
+  template: `<div id="qr-code-full-region"></div>`,
+  mounted () {
+    const config = {
+      fps: this.fps,
+      qrbox: this.qrbox,
+    };
+    const html5QrcodeScanner = new Html5QrcodeScanner('qr-code-full-region', config);
+    html5QrcodeScanner.render(this.onScanSuccess);
+  },
+  methods: {
+    onScanSuccess (qrCodeMessage) {
+      this.$emit('result', qrCodeMessage);
     }
+  }
 });
 ```
 
-### And place it in the DOM as component
-```html
-<qrcode-scanner
-    v-bind:qrbox="250" 
-    v-bind:fps="10" 
-    style="width: 500px;">
-</qrcode-scanner>
-```
+### Usage
+
+```vue
+<template>
+  <qrcode-scanner
+    :qrbox="250" 
+    :fps="10" 
+    style="width: 500px;"
+    @decodedQrCode="onScan"
```

Comment:
I don't see how `@decodedQrCode` is connected. Should the `onScanSuccess` emit `decodedQrCode` as event?

Reply:
Hi you are right, the emitted event name was `result` so it should be `@result`. I chose the word "result" to make it more explicit and easy to understand. I updated the PR :)